packages/testcafe-models/stepper/index.ts

```diff
@@ -0,0 +1,24 @@
+import type { WidgetName } from '../types';
+import Widget from '../internal/widget';
+import Step from './item';
+
+const CLASS = {
+  item: 'dx-stepper',
+};
+export default class Stepper extends Widget {
+  public static className = '.dx-step';
+
+  itemElements: Selector;
+
+  constructor(id: string | Selector) {
+    super(id);
+    this.itemElements = this.element.find(`.${CLASS.item}`);
```

Comment:
[suggestion] in 'internal' folder create 'collection' model and store collections methods inside. In this case we will inherit from 'collection' model. The same approach we can use for 'collectionItem' and avoid code duplicating.